keyboards/gonnerd/keymaps/gam3cat/keymap.c

```diff
@@ -242,34 +230,27 @@ void matrix_init_user(void) {
     #endif
 }
 
-// Runs constantly in the background, in a loop.
 void matrix_scan_user(void) {
```

Comment:
It may actually be better to use `layer_state_set_user` here. 

IIRC, backlight_level may write to EEPROM, and since there is a limited number of write cycles.... this could be bad for your MCU in the long term.

Reply:
Thanks, didn't realize layer_state_set_user existed :)

Reply:
And btw, I would have assumed this if/else below would have avoided the concern you mentioned on MCU lifetime as I had current layer as a global so the backlight_level should only be called on layer changes... basically the same thing layer_state_set_user seems to do.  changed it anyway

    if (current_layer == layer) {
    }
    else {